examples/java/src/main/java/org/apache/beam/examples/complete/datatokenization/transforms/DataProtectors.java

```diff
@@ -247,10 +203,10 @@ private void processBufferedRows(Iterable<Row> rows, WindowedContext context) {
       } catch (Exception e) {
         for (Row outputRow : rows) {
           context.output(
-              failureTag,
-              FailsafeElement.of(outputRow, outputRow)
-                  .setErrorMessage(e.getMessage())
-                  .setStacktrace(Throwables.getStackTraceAsString(e)));
+                  failureTag,
```

Comment:
Is indentation correct here?

Reply:
I think yes, It was before my changes, but i'll check it better.